### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by ericdevries**
Fixes DD-1153

# Description of changes

# How to test

# Related PRs
(Add links)
*

# Notify
@DANS-KNAW/dataversedans

### CODE REVIEW

File: src/main/assembly/dist/cfg/config.yml [hunk 1]
```diff
@@ -11,12 +11,25 @@ server:
     - type: http
       port: 20341
 
-dataverseDatabase:
+database:
```
Comment (janvanmansum) [inline src/main/assembly/dist/cfg/config.yml]: Why did you rename this? This service only works with Dataverse, hence`dataverseDatabase`.

**Review by janvanmansum — CHANGES_REQUESTED** [review]
* Please, fix failing build.
* I have only one further comment so, far. I'll review/test futher whent he build has been fixed.

File: src/main/assembly/dist/cfg/config.yml (hunk 1 above)
Comment (ericdevries) [reply src/main/assembly/dist/cfg/config.yml]: I thought you made a mistake, but I renamed it to match your original intentions again

**Review by janvanmansum — CHANGES_REQUESTED** [review]
* Make sure there is no log output in the build log from the unit tests.
* Use the port number 20340-20341 as specified in https://github.com/DANS-KNAW/dd-dtap/blob/master/docs/micro-service-port-numbers.md

File: docs/index.md [hunk 2]
```diff
@@ -69,3 +69,44 @@ Maven's `-P` switch: `mvn -Pprm install`.
 Alternatively, to build the tarball execute:
 
     mvn clean install assembly:single
+
```
Comment (janvanmansum) [inline docs/index.md:72]: The section is only of interest for developers. Move i to a separated page called `dev.md`. See: https://dans-knaw.github.io/dans-module-archetype/common-practices/#documentation

Also, my first choice would be to connect to the vagrant box used in Data Station development. That said, the Docker approach is still useful as an alternative.

File: pom.xml [hunk 3]
```diff
@@ -49,6 +49,55 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.mindrot</groupId>
+            <artifactId>jbcrypt</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-auth</artifactId>
+        </dependency>
+        <dependency>
```
Comment (janvanmansum) [inline pom.xml:68]: Is this used? If not, remove the hsqldb related stuff.

**Review by janvanmansum — CHANGES_REQUESTED** [review]

**Comment by janvanmansum** [comment]
@ericdevries : I'll merge this and address the comments myself.
